src/FillArrays.jl

```diff
@@ -324,7 +324,39 @@ cumsum(x::AbstractFill{Bool}) = cumsum(convert(AbstractFill{Int}, x))
 
 unique(x::AbstractFill{T}) where T = isempty(x) ? T[] : T[getindex_value(x)]
 allunique(x::AbstractFill) = length(x) < 2
-           
+
+
+#########
+# any/all/isone/iszero
+#########
+
+function isone(AF::V) where {V <: AbstractFill{T,2}} where T
```

Comment:
This signature is hard to read. This is preferable:
```julia
function isone(AF::AbstractFill{<:Any,2})
```

Reply:
Yes, that's better.